File: C#/Solution #1  O(2^n) Time & O(n) Space.cs

// O(2^n) time | O(n) space

​public class Program {

	public static int GetNthFib(int n) {
		if (n == 2) {
			return 1;
		} else if (n == 1) {
			return 0;
		} else {
			return GetNthFib(n - 1) + GetNthFib(n - 2);
		}
	}
}

File: Swift/Solution #1  O(2^n) Time & O(n) Space.swift

​// O(2^n) time | O(n) space

func getNthFib(n: Int) -> Int {
    if n == 2 {
        return 1
    } else if n == 1 {
        return 0
    } else {
        return getNthFib(n: n - 1) + getNthFib(n: n - 2)
    }
}
